stingray/gti.py

```diff
@@ -328,3 +328,70 @@ def get_btis(gtis, start_time=None, stop_time=None):
 def gti_len(gti):
     """Return the total good time from a list of GTIs."""
     return np.sum([g[1] - g[0] for g in gti])
+
+def check_separate(gti0, gti1):
+    """
+    Check if two GTIs are mutually exclusive.
+
```

Comment:
The docstring should describe more precisely what it actually does. We are not really testing if they are mutually exclusive, but if the two observations overlap. If we had an interval of the second GTI list in the middle of the other GTIs, the GTIs would be mutually exclusive but this function would return `False`.